### PR TITLE
fix batchnorm freeze mean and var

### PR DESCRIPTION
**Pull request by roywei**
As per user feedback, Keras-MXNet was not able to freeze the running mean and var weights during fine tune.

This PR fix the bug and added unit test
Reference:
https://discuss.mxnet.io/t/how-freeze-batchnorm-layer-in-symbolblock/3949

### CODE REVIEW

**Comment by sandeep-krishnamurthy** [comment]
Thanks.
